tests/test_professional_consultation_guard.py

```
@@ -0,0 +1,301 @@
+
+    install_professional_consultation_guard()
+
+    assert getattr(StableLegalProductionService, "_korgan_professional_consultation_guard", False) is True
+    assert StableLegalProductionService.consult.__module__ == "korgan.professional_consultation_guard"
+
+
+def test_guard_runs_through_actual_production_service_chain(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    install_professional_consultation_guard()
+    monkeypatch.setattr(guard, "_corpus_article_check", lambda *args: True)
```

Ruff (0.16.1)

[warning] Unused lambda argument: `args`

(ARG005)